client/file-browser/get-files.js

```diff
@@ -2,7 +2,13 @@ import { promises as fsPromises } from 'fs'
 import path from 'path'
 
 export default async function readFolder(folderPath) {
-  const names = await fsPromises.readdir(folderPath)
+  const names = await fsPromises.readdir(folderPath).catch(err => {
```

Comment:
This is inside an `async` function and already being `await`ed so there's no real need to use `.catch` here, would prefer to just use try/catch, i.e.
```js
let names
try {
  names = await fsPromises.readdir(folderPath)
} catch (err) {
  // ...
}